dom0/sd-workstation.top

```diff
@@ -21,13 +21,11 @@ base:
     - sd-remove-unused-templates
 
   sd-small-bullseye-template:
-    - match: pcre
```

Comment:
yup, no longer needed since there isn't any regex, but very cool advanced salt feature that @eaon found!